### ISSUE TITLE
should Decreased activity of complement receptor [HP:0025541] be 'decreased activation'?

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#3329]
The term _Decreased activity of complement receptor_ [HP:0025541] is missing a definition and is the child of _Abnormality of complement system_.

I wonder if this term should be renamed to decreased activation of the complement system? Or could that be a synonym?

cc @LCCarmody

**Comment by pnrobinson** [comment]
There are no annotated diseases at present.  There are also multiple kinds of complement receptor. I think this part of the HPO might need revision. Please do not change the term for now, let us try to find out what the original use case was. @drseb any idea? Otherwise let us research what types of complement receptor activity are measured clinically.

**Comment by drseb** [comment]
Sorry. No idea. You created it June 2017. Workshop?

**Comment by LCCarmody** [comment]
This came from the Immunology Hackathon.  The notes don’t give the term too much context:
“Complement
Kate to send file
Added term abnormal complement receptor activity”  https://docs.google.com/document/d/1VWEj0z-bleRoiw5j25jTHYPIIwFF9RvkpjcR3nVfH_U/edit?usp=sharing

So we added it during the hackathon.

Not sure this helps Peter jog his memory;  It didn’t help me and I was there. :/




--
Leigh Carmody, PhD
Scientific Curator, Robinson Lab
The Jackson Laboratory for Genomic Medicine
Farmington, CT
(860) 837-2040
leigh.carmody@jax.org<mailto:leigh.carmody@jax.org>
www.jax.org<http://www.jax.org/>

The Jackson Laboratory: Leading the Search for tomorrow’s cures

**Comment by nicolevasilevsky** [comment]
Looking into this term further, it looks like there is a [GO term for 'complement receptor activity'](https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004875), so this is probably the correct label.

I'll add the text def and logical def to this term.

**Comment by nicolevasilevsky** [comment]
actually, I need the GO term to MIREOTed. 
@drseb could you please MIREOT GO:0004875 complement receptor activity

**Comment by nicolevasilevsky** [comment]
For future reference, the logical def should be:
'has part' some 
('decreased functionality' and ('inheres in' some 'complement receptor activity') and ('has modifier' some abnormal))

**Comment by drseb** [comment]
done @nicolevasilevsky

**Comment by pnrobinson** [comment]
I still do not think this is a valid term. In clinical medicine, sometimes also people use words like "activity" to mean activity in an elisa assay and are really ta;lking about concentration.  In any case
'has part' some
('decreased functionality' and ('inheres in' some 'complement receptor activity') and ('has modifier' some abnormal))
is not something that can be measured clinically and this term is not correct.  Let us please research what was really meant by this NTR, it is still non obvious to me!

**Comment by pnrobinson** [comment]
Probably this was related to designate CD59 deficiency and DAF (CD55 deficiency)  (Prof., Sullivan)
--let's refactor

**Comment by pnrobinson** [comment]
Note we already have a term for Reduced fibroblast CD59 level, but this is not the same as complement receptor

**Comment by pnrobinson** [comment]
I cannot figure out this term and will obsolete/merge it.
